handlers/handler.go

```diff
@@ -95,6 +96,56 @@ func (env *Env) PostUserHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(reqUser)
 }
 
+// GetUserHandler gets a user returning specified columns
+func (env *Env) GetUserHandler(w http.ResponseWriter, r *http.Request) {
+	userID, err := strconv.ParseInt(r.Header.Get("User-ID"), 10, 64)
+	if err != nil {
+		errMsg := "Invalid user ID"
+		log.Println(errMsg + ": " + err.Error())
+		http.Error(w, errMsg, http.StatusBadRequest)
+		return
+	}
+
+	user, err := env.DB.ReadUser(userID)
+	if user == nil {
+		errMsg := "User-ID not found"
```

Comment:
Should just be `User not found`.
```suggestion
		errMsg := "User not found"
```